zonemaster/zonemaster-gui
# Translate pagination fr

## Purpose

Provides French translation for pagination component.

## Context

Based on #405 

## Changes

Update translation strings for labels in pagination component

## How to test this PR

* Set the language to French
* Open history on a domain
* Using the accessibility inspector check that the ««, «, », »» buttons have an translated alt text.
